modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -77,6 +79,20 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		HttpServletRequest httpServletRequest, String name) {
+
+		CookieUtil.get(httpServletRequest, "test123");
+		CookiesManagerUtil.getCookieValue(name, httpServletRequest);
+	}
+
+	public void method(
+		HttpServletRequest httpServletRequest, String name, boolean toUpperCase) {
+
+		CookieUtil.get(httpServletRequest, "test123", toUpperCase);
```

Comment:
@MicaelleSilvaa, I checked your [answer in the previous pull](https://github.com/natocesarrego/liferay-portal/pull/1404#discussion_r1386871709), however it seems to be a valid case. For example, consider the scenario in which the developer used the method `CookieUtil.get` using a String value directly, like `"test"`, instead of storing the value in a String variable, as `name`, shouldn't it be a valid scenario? At this moment, how the `UpgradeCatchAll` is handling this case? Is it displaying an error message?

Reply:
Resolved :)